app/views/events/badges.html.erb

```diff
@@ -0,0 +1,25 @@
+<h1><%= t('events.badges.title') + "'" + @event.name + "'" %></h1>
+<h3><%= t('events.badges.participants') %></h3>
+
+<%= form_for event_badges_path(@event), :html => { :class => "form-horizontal event" } do |f| %>
+<table>
+  <tr>
+    <th><%= t('events.badges.printName') %></th>
```

Comment:
It's probably a bad idea to have duplicate translations in the yaml, and "name" is already defined as an attribute of `Profile`. You can use `Profile.human_attribute_name(:name)` instead.